EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -21,64 +24,72 @@
     EasyInventoryService easyInventoryService;
     //Endpoints for the categories
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
     //Retrieves all the categories from the database
-    @RequestMapping(value="/category/all", method = RequestMethod.GET)
+    @GetMapping(value="/category/all")
```

Comment:
Changed @RequestMapping to @GetMappint to make the code more readable

Reply:
Good.... less verbose.